.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+    - id: no-commit-to-branch
+      args: [--branch, master]
+    - id: trailing-whitespace
+    - id: end-of-file-fixer
+    - id: check-yaml
+    - id: debug-statements
+  - repo: https://github.com/psf/black
+    rev: 21.12b0
```

Comment:
Might be useful to put the pinned black version into the docs as well somewhere or mention it...

Reply:
Ehhh, it'll be updated periodically. It should be sufficiently self-documented. I briefly put in the docs that you should be using pre-commit for formatting, and it'll automatically update the plugins when they change.

I'll put something in the readme though, that's a good idea